ulauncher/modes/extensions/extension_cli_handlers.py

```diff
@@ -130,3 +133,143 @@ def upgrade_extensions(_: ArgumentParser, args: Namespace) -> bool:
 
     logger.info("\n%s extensions updated", len(updated_extensions))
     return True
+
+
+def preview_extension(_: ArgumentParser, args: Namespace) -> bool:  # noqa: PLR0911, PLR0912, PLR0915 - intentionally single function for ease of maintenance
+    """
+    Run an extension in preview mode (without installing it).
+    Returns True on success, False otherwise.
+    """
+    # Check if Ulauncher is running
+    if not check_app_running(app_id):
+        logger.error("Error: Ulauncher is not running. Please start it first.")
```

Comment:
Nit: I don't like "Please start it first". Maybe "Error: Ulauncher needs to be running in order to preview extensions." or something.

Reply:
Fixed